pkg/networkmanager/dialogs-common.jsx

```diff
@@ -198,10 +201,25 @@
         if (type == 'vlan') settings = getVlanGhostSettings();
         if (type == 'team') settings = getTeamGhostSettings({ newIfaceName });
         if (type == 'bridge') settings = getBridgeGhostSettings({ newIfaceName });
+        if (type == 'wireguard') settings = getWireGuardGhostSettings({ newIfaceName });
     }
 
     const properties = { connection: con, dev, settings };
 
+    async function resolveDeps(type) {
+        if (type === 'wireguard') {
+            try {
+                await cockpit.script("command -v wg");
+            } catch {
+                const os_release = await read_os_release();
+
+                // RHEL/CentOS 8 does not have wireguard-tools
+                if (os_release.PLATFORM_ID !== "platform:el8")
+                    await install_dialog("wireguard-tools");
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19024-20230711-114115-3bc11384-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)